microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md

```diff
@@ -0,0 +1,388 @@
+---
+title: Build PWA-driven Widgets
+description: How to build lightweight PWA-driven Widgets that integrate in the operating system widget dashboard.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: pwa
+ms.date: 08/01/2022
+---
+# Build PWA-driven Widgets
```

Comment:
```suggestion
# Build PWA-driven widgets
```
often each instance is a judgment call per context for natural transparent result